core/market/decentralized/migrations/2020-05-15-093513_init-db/up.sql

```diff
@@ -11,6 +11,13 @@ CREATE TABLE market_offer (
 	expiration_ts DATETIME NOT NULL
 );
 
+CREATE TABLE market_offer_unsubscribed (
+    id VARCHAR(97) NOT NULL PRIMARY KEY,
+    insertion_ts DATETIME NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    expiration_ts DATETIME NOT NULL,
+    node_id VARCHAR(20) NOT NULL
```

Comment:
```suggestion
CREATE TABLE market_offer_archive (
    id VARCHAR(97) NOT NULL PRIMARY KEY,
    properties TEXT NOT NULL,
    constraints TEXT NOT NULL,    
    node_id VARCHAR(20) NOT NULL,
    unsubscription_ts DATETIME,  --  null in case of expiration
    insertion_ts DATETIME NOT NULL DEFAULT CURRENT_TIMESTAMP,
    expiration_ts DATETIME NOT NULL,
```